app/src/main/java/tech/zafrani/companionforpubg/fragments/NewsFragment.java

```diff
@@ -0,0 +1,69 @@
+package tech.zafrani.companionforpubg.fragments;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import butterknife.BindView;
+import tech.zafrani.companionforpubg.R;
+import tech.zafrani.companionforpubg.adapters.NewsAdapter;
+import tech.zafrani.companionforpubg.models.NewsItem;
+import tech.zafrani.companionforpubg.utils.PUBGNewsFetch;
+
+
+public class NewsFragment extends BaseFragment implements PUBGNewsFetch.PUBGNewsListener {
+    public static final String TAG = NewsFragment.class.getSimpleName();
+
+    @BindView(R.id.fragment_news_recyclerview)
+    public RecyclerView recyclerView;
+
+    @Nullable
+    private NewsAdapter adapter;
+    @Nullable
+    public PUBGNewsFetch.PUBGNewsListener delegate;
+
+    // region BaseFragment
+
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    protected int getLayoutRes() {
+        return R.layout.fragment_news;
+    }
+
+    @Override
+    public void onViewCreated(@NonNull final View view,
+                              @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        recyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
+        adapter =  new NewsAdapter();
+        recyclerView.setAdapter(adapter);
+
+        final PUBGNewsFetch newsFetch = new PUBGNewsFetch();
+        newsFetch.delegate = this;
+        newsFetch.execute();
+    }
+
+    @Override
+    public void onDestroy() {
+        super.onDestroy();
```

Comment:
before calling super you should check if the recyclerview is null, if not  call `recyclerView.setAdapter(null)` reduce any chances of a memory leak.